tests/test_sessions.py

```diff
@@ -344,6 +347,55 @@ def test_run_always_install_only(self, caplog):
 
         assert session.run_always(operator.add, 23, 19) == 42
 
+    @pytest.mark.parametrize(
+        (
+            "interrupt_timeout_setting",
+            "terminate_timeout_setting",
+            "interrupt_timeout_expected",
+            "terminate_timeout_expected",
+        ),
+        [
+            (None, None, 0.3, 0.2),
```

Comment:
If None is passed, None is the expected value (see above).

For the defaults, pass a different sentinel instead of None and check for that further down. For example, the string "default" would work.